db/migrate/20250724082513_add_new_roles_to_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddNewRolesToUsers < ActiveRecord::Migration[8.0]
+  def change
+    add_column :users, :role, :integer
```

Comment:
```suggestion
    add_column :users, :role, :integer, null: false, default: 0
```

I think we should avoid having null value, and make users `reader`s as default. And made existing users as `writer`s